### ISSUE TITLE
Almost working.. uploading to App Store but listed as "Incomplete"

### ISSUE THREAD

**Issue by jpage4500** [BrantApps/gradle-amazon-app-store-publisher#6]
@BrantApps - sorry to bug you - I figured out how to install this plugin and it's almost working... I really hate updating my app on the Amazon App Store so this will be a real time saver if I can get it to work.

First, off the gradle task works:

```
> Task :publishToAmazonAppStore
 ️🔒 Authenticating...
 ️↕️️ Replacing edit...
 ️🆕️ Creating new edit...
⬅️ Remove APKs from previous edit...
⏫ Uploading new APK(s)...
🎉 New APK(s) published to the Amazon App Store...
```

However, when I view the app on the Developer page it shows up as "Incomplete"
![image](https://user-images.githubusercontent.com/1895553/132032988-30cfff16-3872-402f-82fe-4fd264ef08ef.png)

The description tab doesn't have any release notes filled in:
![image](https://user-images.githubusercontent.com/1895553/132033104-174f5b07-1f2c-4ad7-b7d6-c1f35e8a9a17.png)

Obviously this makes sense as I don't see any way to provide release notes to this library.. I know the triple-t one for the play store uses a file but I didn't see any option like that here. Is this even possible to set via the library?

The other, bigger, issue is the APK doesn't seem valid
![image](https://user-images.githubusercontent.com/1895553/132033607-16e70cfe-fa50-4e25-8de7-c5967ccc267a.png)

I validated the APK that I'm referencing is good and loaded it on a device. It's a full/signed release-build too.. the same one I'd manually upload before. Any ideas why it's not showing up correctly?

**Comment by BrantApps** [comment]
Not at all. The release note bit yep I didn't code in but will look at it. The APK upload is bemusing. I'm in my own upgrade hell with my android app at the moment so you building a valid APK is a tonne better than me right now. I'll try and find some time over the weekend to see what I can see. I appreciate your debugging!

**Comment by jpage4500** [comment]
thanks for the quick reply!

FWIW I think I see why the uploaded APK isn't working - it's not replacing the existing one but adding a new apk (see above screenshot). I see this property for the driver.. should it be 'true'? I'd think it should be but maybe I'm not understanding what it's for.

```
    replaceEdit = true
```
